__tests__/app.test.js

```diff
@@ -89,3 +89,57 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("GET:200 responds with an array of comment objects for the given article_id in descending date order", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then((response) => {
+        const commentsArr = response.body;
+        console.log(response.body)
+        const desiredArr = [
+          {
+            article_id: 3,
+            author: "icellusedkars",
+            body: "Ambidextrous marsupial",
+            comment_id: 11,
+            created_at: "2020-09-19T23:10:00.000Z",
+            votes: 0,
+          },
+          {
+            article_id: 3,
+            author: "icellusedkars",
+            body: "git push origin master",
+            comment_id: 10,
+            created_at: "2020-06-20T07:24:00.000Z",
+            votes: 0,
+          },
+        ];
+        expect(commentsArr).toEqual(desiredArr)
+        commentsArr.forEach((comment) => {
+          expect(comment.article_id).toBe(3);
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.created_at).toBe("string");
+        });
+      });
+  });
```

Comment:
So the way you've written this test actually does make sure, that the more recent comment will be shown first, but I would argue that this is not necessarily the most readable way of testing for that. You don't need to change it, but just something to keep in mind!